tests/queries/0_stateless/00157_cache_dictionary.sql

```diff
@@ -9,7 +9,7 @@ ORDER BY (CounterID, EventDate, intHash32(UserID))
 SAMPLE BY intHash32(UserID)
 SETTINGS storage_policy = 'default';
 
-INSERT INTO test.hits_1m SELECT * FROM test.hits LIMIT 1000000
+INSERT INTO test.hits_1m SELECT * FROM test.hits LIMIT 500000
```

Comment:
Hm, but why does it sometimes takes 10 mins, while usually takes ~30 secs?

https://play.clickhouse.com/play?user=play#U0VMRUNUIGF2Zyh0ZXN0X2R1cmF0aW9uX21zKSwgcXVhbnRpbGVFeGFjdCgwLjkpKHRlc3RfZHVyYXRpb25fbXMpCkZST00gY2hlY2tzCldIRVJFIGNoZWNrX3N0YXJ0X3RpbWUgPj0gbm93KCkgLSBJTlRFUlZBTCAyNCBob3VyCiAgICBBTkQgKGhlYWRfcmVmID0gJ21hc3RlcicgQU5EIHN0YXJ0c1dpdGgoaGVhZF9yZXBvLCAnQ2xpY2tIb3VzZS8nKSkKICAgIEFORCB0ZXN0X3N0YXR1cyAhPSAnU0tJUFBFRCcKLy8gICAgQU5EICh0ZXN0X3N0YXR1cyBMSUtFICdGJScgT1IgdGVzdF9zdGF0dXMgTElLRSAnRSUnKSAKLy8gICAgQU5EIGNoZWNrX3N0YXR1cyAhPSAnc3VjY2VzcycKICAgIEFORCBwb3NpdGlvbih0ZXN0X25hbWUsICcwMDE1N19jYWNoZV9kaWN0aW9uYXJ5JykgPiAw

Reply:
thanks for the info, the root cause should come from index granularity, fixed now.